docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -70,115 +72,167 @@ ifdef::qs[]
+.. *Message*: Type the content of the message that you want to send to the topic, for the quick start example, type `Hello World!`.
+.. *Period*: Specify the interval (in milliseconds) at which you want the Connectors instance to sends messages to the Kafka topic. For the quick start example, specify `10000`, to send a message every 10 seconds.
 
-. In this step you provide the connector specific configuration.
-.. Leave the *Data shape Format* to `application/octet-stream`
-.. *Topic Names*: Enter the name of the topic you created in the previous task of this quick start.
-.. *Content Type*: Leave to `text/plain`.
-.. *Message*: Enter the content of the message that you want to send to the topic, for example `Hello World!`.
-.. *Period*: The interval in milliseconds at which events will be sent. Set this to `10000`, which will produce an event every 10 seconds.
+. Optionally, configure the error handling policy for your Connectors instance. For the quick start, select *log* (the Connectors instance sends errors to its log).  +
++
+Other options are *stop* (the Connectors instance shuts down in case of errors), or *dead letter queue* (the Connectors instance sends messages that it cannot handle to a dead letter topic that you define for the Connectors Kafka instance). 
 
-. Finally configure the error handler for the connector. You can choose between *stop* (the connector shuts down in case of errors), *log* (the error is logged) or *dead letter queue* (the events that cannot be handled by the connector are sent to a dead letter topic of your Kafka instance).
-Select `log`.
+.. Click *Next*.
 
-. The next screen shows the configuration properties of your connector. Click *Create connector* to proceed with the deployment of the connector.
+.. Review the summary of the configuration properties of your Connectors instance and then click *Create Connectors instance* to deploy it.
++
+Your Connectors instance is listed in the table of Connectors. After a couple of seconds, the status of your Connectors instance changes to the *Ready* state and it starts producing messages and sending them to its associated Kafka topic.
 +
-After you complete the connector setup, the new connector is listed in the connectors table. After a couple of seconds, the connector moves to the *Ready* state. At this point the connector starts producing messages to the Kafka topic associated with the connector.
+From the connectors table, you can stop, start and delete your Connectors instance, as well as edit its configuration by clicking the options icon (three vertical dots).
 +
-From the connectors table, you can stop, start and delete the connector, as well as edit the connector configuration by clicking the options icon (three vertical dots).
+In the next procedure, you can verify that the source Connectors instance is sending messages as expected by creating a sink Connectors instance that consumes the messages.
 
 .Verification
```

Comment:
Same as last procedure: make the verification a bit more specific about what exactly the user should verify, and add a non-quick start verification.